modules/provision/remote_store.go

```diff
@@ -45,17 +45,22 @@ func (s *HTTPStore) Reserve(r *Reservation, nodeID modules.Identifier) error {
 }
 
 // Poll retrieves reservations from BCDB. If all is true, it returns all the reservations
-// for this node, otherwise is return only the reservation never sent yet and do long polling
-func (s *HTTPStore) Poll(nodeID modules.Identifier, all bool) ([]*Reservation, error) {
+// for this node.
+// otherwise it returns only the reservation never sent yet or the reservation that needs to be deleted
+// and do long polling
+func (s *HTTPStore) Poll(nodeID modules.Identifier, all bool, since int64) ([]*Reservation, error) {
```

Comment:
You should use a `Time` object or even a `Duration` instead of an int64. It's not clear what the value of since should be.